src/server/log.js

```diff
@@ -14,7 +14,6 @@ function error(error) {
     }
     console.log(chalk.red.bold('SIM ERROR: ' + error));
     config.server && config.server.close();
-    process.exit(1);
```

Comment:
Oh, right - that code was from when this was CLI only. Regarding removing server.close - who will handle closing the server in the API scenario?